app/Resources/static/themes/material/js/shortcuts/entry.js

```diff
@@ -0,0 +1,22 @@
+import Mousetrap from 'mousetrap';
+import $ from 'jquery';
+
+/* open original article */
+Mousetrap.bind('o', () => {
+  $('ul.side-nav li:nth-child(2) a i')[0].click();
+});
+
+/* mark as favorite */
+Mousetrap.bind('s', () => {
+  $('ul.side-nav li:nth-child(5) a i')[0].click();
+});
+
+/* mark as read */
+Mousetrap.bind('a', () => {
+  $('ul.side-nav li:nth-child(4) a i')[0].click();
+});
+
+/* delete */
+Mousetrap.bind('del', () => {
+  $('ul.side-nav li:nth-child(6) a i')[0].click();
```

Comment:
Can't we use custom class here instead of number of items in the list?
I'm sure that if we add a new item in the menu we'll forget about shortcut and generate plenty of problem.